pydarn/plotting/rtp.py

```diff
@@ -0,0 +1,438 @@
+# Copyright (C) 2019 SuperDARN
+# Author: Marian Schmidt
+# This code is improvement based on rti.py in the DaVitpy library
+# https://github.com/vtsuperdarn/davitpy/blob/master/davitpy
+
+"""
+Range-time Intensity plots
+"""
+import matplotlib.pyplot as plt
+from matplotlib import dates, colors, cm, ticker
+import numpy as np
+from typing import List
+from datetime import datetime, timedelta
+
+from pydarn import DmapArray, DmapScalar, dmap2dict, rtp_exceptions
+
+
+class RTP():
+    """
+    Range-time intensity plots SuperDARN data using the following fields:
+
+    Class pattern design: Builder
+    This class inherits matplotlib.pyplot to inherit plotting features as well
+    build off their builder design pattern.
+
+    Notes
+    -----
+    This class inherits from matplotlib to generate the figures
+
+    Methods
+    -------
+    plot_profile
+    plot_scalar
+    summary_plot
+
+    """
+
+    @classmethod
+    def plot_range_time(cls, dmap_data: List[dict], *args,
+                        parameter: str = 'p_l', beam_num: int = 0, ax=None,
```

Comment:
Stylistically, I recommend removing the whitespace about the '=' here

Reply:
Flake 8 PEP 8 linter disagrees. I am just going to follow Flake 8 because I encourage others to use it to find syntax errors and PEP 8 violations.